templates/AureliaSpa/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet:latest
```

Comment:
This file is slightly out of date (compared with the other templates). Would you be able to update it, e.g., based on https://github.com/aspnet/JavaScriptServices/blob/dev/templates/Angular2Spa/Dockerfile?